### ISSUE TITLE
discogs_client.tests.test_utils.UtilsTestCase.test_sort fails on Python 3.13

### ISSUE THREAD

**Issue by wRAR** [joalla/discogs_client#156]
```
FAIL: test_sort (discogs_client.tests.test_utils.UtilsTestCase.test_sort)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "discogs_client/discogs_client/tests/test_utils.py", line 76, in test_sort
    self.assertEqual(utils.Sort.By.ARTIST, 'artist')
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: <By.ARTIST: 'artist'> != 'artist'
```

Found in https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1082206 and I can confirm it locally.

**Comment by JOJ0** [comment]
@wRAR this is a very old version of python3-discogs-client and I'm wondering how the official way to remove a package from the Debian archives would be? We don't want to maintain the Debian package anymore. Can you help us with that matter?

**Comment by wRAR** [comment]
@JOJ0 the official way to remove a package from Debian unstable/testing is making sure there are no reverse dependencies (there are currently two direct ones) and then filing an RM bug.

**Comment by JOJ0** [comment]
@wRAR thanks a lot, sorry for late reply, I should finally get to removing it. One dependency is github.com/joj0/discodos.git, not sure about the second yet.

**Comment by wRAR** [comment]
`beets` uses `python3-discogs-client` for build-time tests and autopkgtests.

**Comment by JOJ0** [comment]
@wRAR Hi sgain I'm so sorry that I'm so slowtp respond/act.

I justrealized that python-discogs-client is somewhat maintained in Debian. Who does ot? I see v2.5, which is notterribly old but mist importantly I think we recently fixedsom Pytjon 12 related things which might address the issue:

**Comment by wRAR** [comment]
> I justrealized that python-discogs-client is somewhat maintained in Debian.

I can't say so. If you no longer want to be listed as its maintainer it must be formally orphaned as there are no other human maintainers listed.

> I see v2.5

It was uploaded 2 years ago as a team upload.

**Comment by JOJ0** [comment]
Ok, hi again @wRAR thanks again for the patience. I now did the following:

- Install a Docker machine with some Debian tools (since I don't have a current Debian machine around anymore)
- Use `reportbug` to file an "wnpp orphan" bug for the `python-discogs-client` source package.
- Release a new python3-discogs-client package https://github.com/joalla/discogs_client/releases/tag/v2.7.1 and on PyPI https://pypi.org/project/python3-discogs-client/2.7.1/ (which I believe fixes the mentioned "Debian Python 3.13" bug even though we've seen it in 3.12 already.)

After thinking this through again I believe that the package is essential to stay in Debian since https://packages.debian.org/source/trixie/beets depends on it and I can imagine that having `beets` available out-of-the box on Debian is worth it.

Thanks for pointing me in the right direction @wRAR, is there anything else I can do?

**Comment by wRAR** [comment]
> is there anything else I can do?

No, though I suggest considering doing the same for other your packages you don't want to maintain. Thanks.

**Comment by JOJ0** [comment]
> > is there anything else I can do?
> 
> No, though I suggest considering doing the same for other your packages you don't want to maintain. Thanks.

Yes. Thanks. A little off-topic in this repo but I don't care and I think it's not a big deal for my fellow repo owners if I continue to talks about "other" Debian packages within this issue :-)

I just filed removal requests for the remaining two packages I'm listed as a maintainer. I was not 100% sure if filing against releases.debian.org was correct (since they are in testing already. My goal is to remove them from Debian entirely.

- https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1087545
- https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1087548


 Would I now have to file another bug against `ftp.debian.org` to remove them from unstable/sid? Or is it sufficient like that?

**Comment by wRAR** [comment]
The correct way to remove a package from Debian is to file a removal from sid against `ftp.debian.org`. https://www.debian.org/doc/manuals/developers-reference/pkgs.en.html#removing-packages

**Comment by JOJ0** [comment]
I think this is sorted by now. I'll close this one and thanks again for all the help @wRAR
